### PR TITLE
PCI Kubernetes: Get available flavors from API

### PR DESCRIPTION
**Pull request by marie-j**
### Related 
- [x] https://github.com/ovh-ux/ovh-api-services/pull/258 

ref: MANAGER-3892

### CODE REVIEW

**Comment by antleblanc** [comment]
@marie-j [`ovh-api-services@9.25.0`](https://www.npmjs.com/package/ovh-api-services/v/9.25.0) has been released :tada:

**Comment by antleblanc** [comment]
Hi @marie-j!

Could you please rebase your Pull Request on `develop` branch? (In conflict due to #1789)

Thanks!

File: packages/manager/apps/carrier-sip/package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "moment": "^2.15.2",
     "ng-csv": "^0.3.6",
     "oclazyload": "^1.1.0",
-    "ovh-api-services": "^9.24.1",
+    "ovh-api-services": "^9.25.0",
```
Comment (FredericEspiau) [inline packages/manager/apps/carrier-sip/package.json]: Not up to date :)

File: packages/manager/apps/cloud/package.json [hunk 2]
```diff
@@ -102,7 +102,7 @@
     "ovh-angular-browser-alert": "ovh-ux/ovh-angular-browser-alert#^1.0.0",
     "ovh-angular-list-view": "ovh-ux/ovh-angular-list-view#^0.1.5",
     "ovh-angular-responsive-page-switcher": "^1.1.1",
-    "ovh-api-services": "^9.24.1",
+    "ovh-api-services": "^9.25.0",
```
Comment (FredericEspiau) [inline packages/manager/apps/cloud/package.json]: Not up to date :)

File: packages/manager/apps/dedicated/package.json [hunk 3]
```diff
@@ -114,7 +114,7 @@
     "office-ui-fabric-core": "^11.0.0",
     "ovh-angular-browser-alert": "ovh-ux/ovh-angular-browser-alert#^1.0.0",
     "ovh-angular-responsive-tabs": "^4.0.0",
-    "ovh-api-services": "^9.24.1",
+    "ovh-api-services": "^9.25.0",
```
Comment (FredericEspiau) [inline packages/manager/apps/dedicated/package.json]: Not up to date :)

File: packages/manager/apps/enterprise-cloud-database/package.json [hunk 4]
```diff
@@ -34,7 +34,7 @@
     "lodash": "^4.17.14",
     "messenger": "HubSpot/messenger#~1.4.1",
     "oclazyload": "^1.1.0",
-    "ovh-api-services": "^9.24.1",
+    "ovh-api-services": "^9.25.0",
```
Comment (FredericEspiau) [inline packages/manager/apps/enterprise-cloud-database/package.json]: Not up to date :)

File: packages/manager/apps/carrier-sip/package.json (hunk 1 above)
Comment (marie-j) [reply packages/manager/apps/carrier-sip/package.json]: Yes, another release was made after

**Comment by marie-j** [comment]
> Hi @marie-j!
> 
> Could you please rebase your Pull Request on `develop` branch? (In conflict due to #1789)
> 
> Thanks!

@antleblanc Done ! I've removed the commit responsible for the bump of ovh-api-services which was the one causing conflicts

**Pull request merged**